package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "harmony-marketplace-sdk",
+  "version": "1.0.0",
+  "description": "Harmony Marketplace SDK",
+  "author": "Blockcoders <engineering@blockcoders.io>",
+  "license": "MIT",
+  "readmeFilename": "README.md",
+  "main": "./lib/index.js",
+  "module": "./lib.esm/index.js",
+  "jsnext:main": "./lib.esnext/index.js",
+  "types": "./lib/index.d.ts",
+  "engineStrict": false,
+  "engines": {
+    "node": "^12.x || >=14.x || >=16.x"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "NODE_ENV=test echo \"no test specified\" && exit 0",
+    "test:cov": "NODE_ENV=test echo \"no test specified\" && exit 0",
+    "build": "tsc --build ./tsconfig.json",
+    "build-all": "npm run build-cjs && npm run build-esnext && npm run build-esm",
+    "build-cjs": "tsc -p tsconfig.json --outDir lib -t es3 -m commonjs",
+    "build-esnext": "tsc -p tsconfig.json --outDir lib.esnext -t es2015 -m commonjs",
+    "build-esm": "tsc -p tsconfig.json --outDir lib.esm -t es2015 -m es2015",
+    "lint:ci": "eslint \"{src,apps,libs,test}/**/*.ts\"",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blockcoders/harmony-marketplace-sdk.git"
+  },
+  "keywords": [
```

Comment:
Add nft, hr721, hr1155, harmony-sdk and one.

Reply:
Done!